docs/install_guides/installing-openvino-yocto.md

```diff
@@ -43,26 +43,26 @@ MACHINE = "intel-skylake-64"
 # Enable clDNN GPU plugin when needed.
 # This requires meta-clang and meta-oe layers to be included in bblayers.conf
 # and is not enabled by default.
-PACKAGECONFIG_append_pn-openvino-inference-engine = " opencl"
+PACKAGECONFIG:append:pn-openvino-inference-engine = " opencl"
 
 # Enable building OpenVINO Python API.
 # This requires meta-python layer to be included in bblayers.conf.
-PACKAGECONFIG_append_pn-openvino-inference-engine = " python3"
+PACKAGECONFIG:append:pn-openvino-inference-engine = " python3"
 
 # This adds OpenVINO related libraries in the target image.
-CORE_IMAGE_EXTRA_INSTALL_append = " openvino-inference-engine"
+CORE_IMAGE_EXTRA_INSTALL:append = " openvino-inference-engine"
```

Comment:
Just a question: is "openvino-inference-engine" changed too since inference engine is not used in 2022.1?

Reply:
No, that is just the package name that is produced by Yocto and that is unchanged.